release/release.sh

```diff
@@ -2,6 +2,8 @@ set -ex
 
 rm -rf dist
 
+#export DS_KERNELS_MAKE_JOBS=10
```

Comment:
Any reason to keep this? Perhaps we can add a comment indicating users can uncomment this if they want to control the number?

Reply:
Added a comment.